onnxutils/onnxtk/common/data_types.py

```diff
@@ -111,6 +127,9 @@ def __init__(self, shape=None, doc_string=''):
     def _get_element_onnx_type(self):
         return onnx_proto.TensorProto.STRING
 
+    def __repr__(self):
+        return "StringTensorType(shape={0})".format(self.shape)
```

Comment:
will it cause any exception if shape is None?

Reply:
No, it will just return: 'StringTensorType(shape=None)'

Reply:
Thanks for the review!